applications/chats/services.py

```diff
@@ -0,0 +1,35 @@
+from django.core.exceptions import ValidationError
+from django.db import transaction
+from django.utils.translation import gettext_lazy as _
+
+from .models import Room
+
+
+class RoomCreator:
+
+    @transaction.atomic
+    def create_from_target_match(self, target, *members):
+        if self._exists_equal_room(target.topic, *members):
+            return None
```

Comment:
If you want just to break out of the function I think it's better to use just `return`. Using `return None` makes it look like this function should be returning something to be used in another place.